server/common/default_config.py

```diff
@@ -1,129 +1,148 @@
 import yaml
 
 default_config = """
```

Comment:
Future task - we need a light weigh document that describes every config parameter, its default, etc.  Not for this PR, but would be good to file an issue and link them.

Reply:
Agreed.  I've started commenting as I add new ones, but there are many that are not commented.